test/instrumentation/testdata/stable-metrics-list.yaml

```diff
@@ -88,6 +88,35 @@
   help: The count of hidden metrics.
   type: Counter
   stabilityLevel: BETA
+- name: certificate_manager_server_rotation_seconds
+  subsystem: kubelet
+  help: Histogram of the number of seconds the previous certificate lived before being
+    rotated.
```

Comment:
This seems like a weird metric... it's measured as the time since it's validity start period to the time it was rotated: https://github.com/kubernetes/kubernetes/blob/7080b51ee92f67623757534f3462d8ae862ef6fe/staging/src/k8s.io/client-go/util/certificate/certificate_manager.go#L595-L597

This seems less useful than measuring the amount of valid time _remaining_ (i.e. how close are we cutting it). It's also not measuring how long the cert was actually in use.

What is the purpose of this metric?